subprojects/soak/build.gradle.kts

```diff
@@ -18,6 +18,7 @@ dependencies {
     integTestImplementation(project(":persistent-cache"))
     integTestImplementation(libs.slf4jApi)
     integTestImplementation(libs.jetty)
+    integTestImplementation(libs.assertj)
```

Comment:
Why do we need assertj in the soak tests? Looks like we didn't need it before, right?

Reply:
Let me rephrase my question: Why do soak tests use `assertj`? I am surprised by this since we have been using Spock for Soak tests as well.

Reply:
I've added the actual upgrade to Spock 2.1 (which is essentially the same as 2.2-M1, but it has the Groovy 4 checks in place). Now we need this change.

Reply:
> Let me rephrase my question: Why do soak tests use `assertj`? I am surprised by this since we have been using Spock for Soak tests as well.

No idea, but let's investigate that separately, shall we?

Reply:
I took a look. Can you add a
```
because "Kotlin Soak tests use AssertJ"
```

to the dependency?

Reply:
Added.